akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -97,6 +110,45 @@ private[akka] object Running {
   import InternalProtocol._
   import Running.RunningState
 
+  import scala.concurrent.duration._
+  import akka.actor.typed.scaladsl.AskPattern._
+
+  private val log = setup.log
+
+  def startReplicationStream(state: RunningState[S], aa: ActiveActive): Unit = {
```

Comment:
move outside the Behavior instance

Reply:
It uses a lot of things from the scope but yes that is safer